### PR TITLE
Add load files

### PR DESCRIPTION
**Pull request by annac02**
Please accept this :) - load script but i am working on the testing

### CODE REVIEW

**Review by MilesDrabwell — COMMENTED** [review]
Gucci

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 pytest
 pymssql
```
Comment (HighestAuto) [inline requirements.txt:2]: pymssql sucks

**Review by HighestAuto — APPROVED** [review]
sure ig, it works. for now

**Review by MilesDrabwell — APPROVED** [review]
This is appropriate review yes

**Review by LinfanS — APPROVED** [review]
List of things to fix: quick fix to the type hint, uncomment the import of the transform, load function needs a return type hint.

**Pull request merged**